README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
I actually like `s.string({min, max})`. This is a pretty well understood notation.

The only reason it's called `s.object(fields)` is that it takes a list of *actual fields*. Maybe that should have been `s.object(properties)`.